.github/workflows/tests.yml

```
@@ -10,15 +10,35 @@ on:
     - cron: '0 0 * * *'
 
 jobs:
-  linux_tests:
+  tests:
     runs-on: ubuntu-22.04
 
     strategy:
       fail-fast: true
       matrix:
-        php: [7.3, 7.4, '8.0', 8.1, 8.2]
+        php: [7.3, 7.4, 8.0, 8.1, 8.2]
+        laravel: [^6.0, ^7.0, ^8.0, ^9.0, ^10.0]
+        exclude:
+          - php: 7.3
+            laravel: ^9.0
+          - php: 7.3
+            laravel: ^10.0
+          - php: 7.4
+            laravel: ^9.0
+          - php: 7.4
+            laravel: ^10.0
+          - php: 8.0
+            laravel: ^10.0
+          - php: 8.1
+            laravel: ^6.0
+          - php: 8.1
+            laravel: ^7.0
+          - php: 8.2
+            laravel: ^6.0
+          - php: 8.2
+            laravel: ^7.0
 
-    name: PHP ${{ matrix.php }}
+    name: P${{ matrix.php }} - L${{ matrix.laravel }}
```

This reads nicer and is how we do it everywhere.

```suggestion
    name: PHP ${{ matrix.php }} - Laravel ${{ matrix.laravel }}
```